turbopack/crates/turbopack-node/js/src/transforms/webpack-loaders.ts

```diff
@@ -301,6 +302,18 @@ const transform = (
               request: string,
               callback?: (err?: Error, result?: string) => void
             ) => {
+              if (path.isAbsolute(request)) {
+                // Relativize absolute requests. Turbopack disallow them in JS code, but here it's
+                // generated programatically and there is a smaller problem of
+                // non-cacheable/non-portable builds.
+                request = path.relative(lookupPath, request)
+                // On Windows, the path might be still absolute if it's on a different drive. Just
+                // let the resolver throw the error in that case.
+                if (!path.isAbsolute(request) && !request.startsWith('.')) {
```

Comment:
It could be a filename starting with .

Reply:
```suggestion
                if (!path.isAbsolute(request) && request.split(path.sep)[0] !== '..') {
```

Just splitting on `path.sep` is fine on Windows: `path.relative` normalizes the `/` to `path.sep`.

> On Windows, both the forward slash (/) and backward slash (\) are accepted as path segment separators; however, the path methods only add backward slashes (\).

https://nodejs.org/api/path.html#pathsep

<img width="445" height="46" alt="Image" src="https://github.com/user-attachments/assets/314929ba-e7f4-4ccc-8a72-7c30dd1851c5" />